hoomd/pytest/test_snapshot.py

```diff
@@ -125,6 +126,131 @@ def test_configuration(s):
         assert s.configuration.dimensions == 3
 
 
+def generate_outside(box, interior_points, multipliers, initial_images):
```

Comment:
There's some inconsistent naming here between `multiples` and `multipliers`. But neither name is clear that it relates to a number of lattice vectors to unwrap. Based on my experience with freud's naming conventions, I would suggest `unwrap_images` instead of `multiples` and `multipliers`, which also parallels the name `initial_images`. Examples of "unwrap":
- https://freud.readthedocs.io/en/latest/modules/box.html#freud.box.Box.unwrap
- https://hoomd-blue.readthedocs.io/en/latest/module-hoomd-write.html?highlight=unwrap#hoomd.write.DCD.unwrap_full
```suggestion
def generate_outside(box, interior_points, unwrap_images, initial_images):
```
...and likewise below. (`factor` would become `unwrap_image`, to parallel `initial_image`.)